docs/sources/tempo/getting-started/example-demo-app.md

```diff
@@ -44,6 +44,12 @@ There are monolithic mode and microservices examples.
 
 To learn how to set up a Tempo cluster, see [Deploy on Kubernetes with Tanka]({{< relref "../setup/tanka" >}}).
 
+## Introduction to Metrics, Logs and Traces example
```

Comment:
TNS demo feels unmaintained. Do you mind just removing the section below? I would much rather point people to intro to mlt